Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -275,9 +275,27 @@ class TabLocationView: UIView {
         if let searchUrl = SearchURL(url) {
             return searchUrl.query
         }
+        if self.isURLHostIPAddress(url: url) {
+            return url.host ?? ""
+        }
         return url.publicSuffix(additionalPartCount: 1) ?? ""
     }
 
+    private func isURLHostIPAddress(url: URL) -> Bool {
```

Comment:
This could be put in url extension - also it should handle IPv6

Reply:
I have put this method in url extension but url.isIPv6 is always returns false.